features/user_can_comment_on_article.feature

```diff
@@ -1,6 +1,6 @@
-Feature: Subscriber can comment on article
+Feature: user can comment on article
 
-  As a Subscriber
+  As a Users
```

Comment:
!! `As a Users`??

Reply:
sorry for the typo, it has been fixed and pushed